arch/x86/crc32_fold_pclmulqdq.c

```diff
@@ -3,7 +3,7 @@
  * instruction.
  *
  * A white paper describing this algorithm can be found at:
- * https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/fast-crc-computation-generic-polynomials-pclmulqdq-paper.pdf
+ * http://intel.ly/2ySEwL0
```

Comment:
I actually think having the original filename of the white paper can be quite useful.
Not sure whether the direct link or the shortened url will be more reliable in the long term, but with the filename it is easier to search for if the link it broken.

Reply:
Removed. I think it is best if we include the pdf in our docs.